tests/unit/testaddon.cpp

```diff
@@ -1147,11 +1147,11 @@ void TestAddon::telemetry_state_change() {
   // Event metrics can only record InvalidValue or InvalidOverflow errors
   // https://mozilla.github.io/glean/book/reference/metrics/event.html#recorded-errors
 
-  QCOMPARE(mozilla::glean::sample::addon_state_changed.testGetNumRecordedErrors(
-               VPNGlean::InvalidValue),
+  QCOMPARE(mozilla::glean::sample::addon_state_changed.numRecordedErrors(
```

Comment:
Please change this back to `testGetNumRecordedErrors`. It is important we don't diverge from the Glean APIs (https://mozilla.github.io/glean/book/reference/metrics/event.html#testgetnumrecordederrors)